### ISSUE TITLE
Remove Nubbin & Close X From Color Picker

### ISSUE THREAD

**Issue by kwilloughby-sf** [salesforce/design-system-react#1724]
The DSR Color Picker should match the SLDS Color Picker (https://www.lightningdesignsystem.com/components/color-picker/).

The items below should be tweaked (this is not an exhaustive list).
- Remove the nubbin from the DSR Color Picker and move the Popover up to match the SLDS Color Picker
- Remove the close x in the upper right corner of the DSR Color Picker.
- Make the DSR Color Picker tab text size larger (i.e. 13px, visual redline from 220 SLDS).
- Make the DSR Color Picker selected tab's blue bar thicker (i.e. 3px, visual redline from 220 SLDS).
- Make the DSR Color Picker gray bar under the tabs thinner (i.e. 1px, visual redline from 220 SLDS).
- Make the DSR Color Picker swatches on the default tab larger (i.e. 20px x 20px, visual redline from 220 SLDS, probably due to gray border). Retain the border for the transparent swatch but make it rgb(201, 199, 197).
- Make the DSR Color Picker selected swatch treatment match on both tabs. Remove all current selected swatch styling and add a 1px black border on the inside of the swatch with a 1px white border on the inside of the black border for all swatches. See examples below.

Image Below Shows Blue Selected
![image](https://user-images.githubusercontent.com/42620858/51944587-6aa49080-23ea-11e9-8e0e-dba6342ab5ad.png)

Image Below Shows Transparent Selected
![image](https://user-images.githubusercontent.com/42620858/51938422-075f3200-23db-11e9-8b19-0eac31601a2e.png)

@interactivellama I may need to go through this and do an engineering spec if anything is not straightforward.

**Comment by davidlygagnon** [comment]
Thanks @kwilloughby-sf - I can take a look at this.

**Comment by thobhanifreddy** [comment]
I would like to work on this issue. 
I have some question, before starting. 

> Remove the close x in the upper right corner of the DSR Color Picker.

can I add prop `closeIcon: PropTypes.bool` in popoverwhich will decide whether to show `x` icon. 

> Remove the nubbin from the DSR Color Picker and move the Popover up to match the SLDS Color Picker

I don't really understand this! :(

> Make the DSR Color Picker tab text size larger (i.e. 13px, visual redline from 220 SLDS).

can I add props `labelSize: PropTypes.number` in tab component? 

> Make the DSR Color Picker selected tab's blue bar thicker 
> Make the DSR Color Picker gray bar under the tabs thinner

I can't really figure this out can anyone help me out with this? 

> Make the DSR Color Picker swatches on the default tab larger 

Looks like this is already done.

**Comment by thobhanifreddy** [comment]
I would like to work on this issue. 
I have some question, before starting. 

> Remove the close x in the upper right corner of the DSR Color Picker.

can I add prop `closeIcon: PropTypes.bool` in popoverwhich will decide whether to show `x` icon. 

> Remove the nubbin from the DSR Color Picker and move the Popover up to match the SLDS Color Picker

I don't really understand this! :(

> Make the DSR Color Picker tab text size larger (i.e. 13px, visual redline from 220 SLDS).

can I add props `labelSize: PropTypes.number` in tab component? 

> Make the DSR Color Picker selected tab's blue bar thicker 
> Make the DSR Color Picker gray bar under the tabs thinner

I can't really figure this out can anyone help me out with this? 

> Make the DSR Color Picker swatches on the default tab larger 

Looks like this is already done.

**Comment by pulkonet** [comment]
I would like to work on this issue

**Comment by interactivellama** [comment]
@kwilloughby-sf These issues should be fixed now http://design-system-react-components.herokuapp.com/?selectedKind=SLDSColorPicker&selectedStory=Default&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel

**Comment by kwilloughby-sf** [comment]
Thanks for the heads up @interactivellama. I do see a few discrepancies between my mockups and Storybook. I will create a new issue. For example, the transparent item should still have a gray border when it's unselected.
